### ISSUE TITLE
New 2015.07.18 fails in Python 2.7

### ISSUE THREAD

**Issue by jamiesonbecker** [rg3/youtube-dl#6274]
Awesome module. here's what happens for me (Python 2.7)

```
Collecting youtube-dl
  Downloading youtube_dl-2015.07.18-py2.py3-none-any.whl (993kB)
    100% |████████████████████████████████| 995kB 159kB/s 
Installing collected packages: youtube-dl
  Found existing installation: youtube-dl 2015.6.15
    Uninstalling youtube-dl-2015.6.15:
      Successfully uninstalled youtube-dl-2015.6.15
Successfully installed youtube-dl-2015.7.18
```
```
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl", line 7, in <module>
    from youtube_dl import main
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/__init__.py", line 16, in <module>
    from .options import (
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/options.py", line 8, in <module>
    from .downloader.external import list_external_downloaders
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/downloader/__init__.py", line 3, in <module>
    from .common import FileDownloader
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/downloader/common.py", line 8, in <module>
    from ..compat import compat_str
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/compat.py", line 82, in <module>
    _asciire = re.compile('([\x00-\x7f]+)') if sys.version_info < (2, 7) else compat_urllib_parse._asciire
AttributeError: 'module' object has no attribute '_asciire'
```

**Comment by dstftw** [comment]
This is already fixed and will be available in the next version.
You can also fix this by updating python to 2.7.4 or later. Or you can use previous version `https://yt-dl.org/downloads/2015.07.07/youtube-dl`.

**Comment by jamiesonbecker** [comment]
Awesome, thanks @dstftw !!
